scenarios/turbine-heat-sensor/src/main/java/io/pravega/turbineheatsensor/TurbineHeatSensor.java

```diff
@@ -438,13 +438,16 @@ public void run() {
                 do {
                     try {
                         final EventRead<String> result = reader.readNextEvent(0);
-                        produceStats.runAndRecordTime(() -> {
-                            return CompletableFuture.completedFuture(null);
-                        }, Long.parseLong(result.getEvent().split(",")[0]), 100);
+                        if (result.getEvent() != null) {
+                            consumeStats.runAndRecordTime(() -> {
+                                return CompletableFuture.completedFuture(null);
+                            }, Long.parseLong(result.getEvent().split(",")[0]), 100);
+                            totalEvents--;
+                        }
                     } catch (ReinitializationRequiredException e) {
                         e.printStackTrace();
```

Comment:
Let's create an issue to remove this print stack trace.

Reply:
Ok, I will create it.

Reply:
https://github.com/pravega/pravega-samples/issues/118